src/user/dumbbell_initialization.F90

```diff
@@ -404,9 +446,26 @@ subroutine dumbbell_initialize_sponges(G, GV, US, tv, depth_tot, param_file, use
         enddo
       endif
     enddo ; enddo
-  endif
-
   if (associated(tv%S)) call set_up_ALE_sponge_field(S, G, GV, tv%S, ACSp)
+ else
+    do j=G%jsc,G%jec ; do i=G%isc,G%iec
+      eta(i,j,1) = 0.0
+      do k=2,nz
+        eta(i,j,k) = eta(i,j,k-1)-h_in(i,j,k-1)
```

Comment:
This expression calculating the interface heights is dimensionally inconsistent (eta and depth_tot have units of Z, while h_in has units of H). This expression could be compared with the dimensionally correct one at line 175, above.